### ISSUE TITLE
[Feature Integration Request]

### ISSUE THREAD

**Issue by gp-saurabh** [healthchecks/healthchecks#423]
It would be cool if it can integrate with Google Chat (Incoming Webhook)

[How to? Here. ](https://developers.google.com/hangouts/chat/how-tos/webhooks)

**Comment by lrobinot** [comment]
Google Chat should be integrated through Webhook, this issue could be closed, no?

**Comment by cuu508** [comment]
Yes, you can get basic notifications working with the Webhook integration.

A "native" integration allows several quality-of-life improvements:

* You don't need to copy-paste and edit JSON snippets when setting up a new integration. 
* No worries about special character escaping. Let's say you are posting JSON document and use the $NAME placeholder in it. If a check's name contains a double quote, the resulting JSON will be invalid and the notifications will likely not work
* Access to more data. For example, you can include check's description in the notification (which is not available as a placeholder for webhooks currently)

**Comment by cuu508** [comment]
Apparently incoming webhooks don't work with personal Google accounts – they only work with Google Workspace.

I don't have a Google Workspace organization. I know there is a 14 day trial I could perhaps use, but the integrations require ongoing maintenance, and I don't want to start another 14 day trial every 6 months or so. If there's a way to test incoming webhooks without me having to create trial or paid Workspace accounts, I'd be interested to know.

cc: @SemihMFX
